docs/system_design.md

```diff
@@ -89,6 +89,11 @@ transcripts for city council meetings, maybe you will write a module that simply
 already high quality transcript back, no need for any processing. These are both valid speech recognition models, it's
 simply that one of them produces transcripts that have 100% confidence.
 
+If your city provides closed caption files for city council meetings, you can write a module that takes the closed caption
+file and produce transcripts. If not all city council meetings have closed caption files, you can use a mixture of speech
+recognition models. That is, if there is a closed caption file, use that to produce transcripts. If there is no closed
+caption file you can use a `sr_models.GoogleCloudSRModel` to produce transcripts from an audio file.
```

Comment:
Great documentation already, maybe add a link or explain the `try`, `catch` of the config file as well.